lib/content/put-stream.js

```diff
@@ -8,15 +8,15 @@ var hasContent = require('./read').hasContent
 var moveFile = require('../util/move-file')
 var path = require('path')
 var pipeline = require('mississippi').pipeline
-var randomstring = require('randomstring')
 var rimraf = require('rimraf')
 var through = require('mississippi').through
 var to = require('mississippi').to
+var uniqueFilename = require('unique-filename')
 
 module.exports = putStream
 function putStream (cache, opts) {
   opts = opts || {}
-  var tmpTarget = path.join(cache, 'tmp', (opts.tmpPrefix || '') + randomstring.generate())
+  var tmpTarget = path.join(cache, uniqueFilename('tmp', (opts.tmpPrefix || '')))
```

Comment:
Additionally, you can drop the `(opts... || '')` thing, since `uniqueFilename` handles `null`/`undefined`/`''` just fine as that argument.